enumeratum-core/src/test/scala/enumeratum/values/ValueEnumSpec.scala

```diff
@@ -10,34 +10,50 @@ import org.scalatest.matchers.should.Matchers
 class ValueEnumSpec extends AnyFunSpec with Matchers with ValueEnumHelpers {
 
   describe("basic sanity check") {
-
     it("should have the proper values") {
       LibraryItem.withValue(1) shouldBe LibraryItem.Book
       LibraryItem.withValue(2) shouldBe LibraryItem.Movie
       LibraryItem.withValue(10) shouldBe LibraryItem.Magazine
       LibraryItem.withValue(14) shouldBe LibraryItem.CD
     }
-
   }
 
   testNumericEnum("IntEnum", LibraryItem)
   testNumericEnum("ShortEnum", Drinks)
   testNumericEnum("LongEnum", ContentType)
+
   testEnum("StringEnum", OperatingSystem, Seq("windows-phone"))
   testEnum("CharEnum", Alphabet, Seq('Z'))
   testEnum("ByteEnum", Bites, Seq(10).map(_.toByte))
-  testNumericEnum("when using val members in the body", MovieGenre)
+
+  testNumericEnum("When using val members in the body", MovieGenre)
   testNumericEnum("LongEnum that is nesting an IntEnum", Animal)
   testNumericEnum("IntEnum that is nested inside a LongEnum", Animal.Mammalian)
   testNumericEnum("Custom IntEnum with private constructors", CustomEnumPrivateConstructor)
 
+  describe("AllowAlias") {
+    it("should be supported") {
+      NumericPrecision.values should contain(NumericPrecision.Integer)
+      NumericPrecision.values should contain(NumericPrecision.Int)
+      NumericPrecision.values should contain(NumericPrecision.Float)
+      NumericPrecision.values should contain(NumericPrecision.Double)
+
+      NumericPrecision.withValue("integer") shouldBe NumericPrecision.Int
```

Comment:
Is this worth checking? The `AllowAlias` trait specifies that it would be undefined which one is returned if there are multiple entries with the same value.

Reply:
It's undefined accross the different case which one is returned, but must be consistent accross executions for a same unchanged case